makefiles/tools/openocd.inc.mk

```diff
@@ -18,3 +18,10 @@ OPENOCD_CONFIG ?= $(BOARDSDIR)/$(BOARD)/dist/openocd.cfg
 # Export OPENOCD_CONFIG to required targets
 OPENOCD_TARGETS = debug% flash% reset
 $(call target-export-variables,$(OPENOCD_TARGETS),OPENOCD_CONFIG)
+
+# Export OPENOCD_ADAPTER_INIT to required targets
+$(call target-export-variables,$(OPENOCD_TARGETS),OPENOCD_ADAPTER_INIT)
+
+# Export OPENOCD_RESET_USE_CONNECT_ASSERT_SRST to required targets
+OPENOCD_TARGETS = flash reset
+$(call target-export-variables,$(OPENOCD_TARGETS),OPENOCD_RESET_USE_CONNECT_ASSERT_SRST)
```

Comment:
No need for a variable here and it overwrites `OPENOCD_TARGETS = debug% flash% reset`

```suggestion
# Export OPENOCD_RESET_USE_CONNECT_ASSERT_SRST to required targets
$(call target-export-variables,flash% reset,OPENOCD_RESET_USE_CONNECT_ASSERT_SRST)
```

Reply:
Note that it should also be `flash% reset` to include `flash-only`